.github/workflows/cli.yml

```diff
@@ -0,0 +1,67 @@
+name: run CLI
+on:
+  push:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  basic-usage:
+    name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+    - name: building
+      run: |
+        npm install ts-node -g
```

Comment:
Ditto, may I ask you to remove this line? And let's track which is failed by this, thank you :)

Reply:
OO

Reply:
removed.

Reply:
After you removed that, the CI is still working, so I think it doesn't depend on ts-node at runtime.